src/plugins/chart_expressions/expression_xy/public/components/__snapshots__/xy_chart.test.tsx.snap

```diff
@@ -1751,6 +1751,13 @@ exports[`XYChart component it renders area 1`] = `
             "getAll": [MockFunction],
           }
         }
+        palettes={
+          KbnPalettes {
+            "get": [Function],
+            "getAll": [Function],
+            "query": [Function],
+          }
+        }
```

Comment:
If you look right above that there is also another thing called `paletteService` how can we replace that and how can we avoid having such similar expressions?

Reply:
Yeah this is may attempt to clarify the usage of the legacy `charts/palette` service as the `paletteService` and the new `@kbn/palettes` as `palettes`.

Previously the legacy palette is called `palettes`, `paletteService`, `paletteRegistry`, etc. 

I think this is something we tackle when we deprecate the legacy charts palette service. For now they like side-by-side.